python/hail/docs/getting_started.rst

```diff
@@ -92,56 +92,80 @@ cluster::
 
     ./gradlew -Dspark.version=2.2.0 shadowJar archiveZip
 
-An IPython shell which can run Hail backed by the cluster can be started with
-the following command, it is important that the Spark located at ``SPARK_HOME``
-has the exact same version as provided to the previous command::
+Python and IPython need a few environment variables to correctly find Spark and
+the Hail jar. We recommend you set these environment variables in the relevant
+profile file for your shell (e.g. ``~/.bash_profile``).
 
-    SPARK_HOME=/path/to/spark/ \
-    HAIL_HOME=/path/to/hail/ \
-    PYTHONPATH="$PYTHONPATH:$HAIL_HOME/build/distributions/hail-python.zip:$SPARK_HOME/python:$SPARK_HOME/python/lib/py4j-*-src.zip" \
-    ipython
+.. code-block:: sh
 
-Within the interactive shell, check that you can initialize Hail by running the
-following commands. Note that you must pass in the existing ``SparkContext``
-instance ``sc`` to the ``hail.init`` function.
+    export SPARK_HOME=/path/to/spark-2.2.0/
+    export HAIL_HOME=/path/to/hail/
+    export PYTHONPATH="$PYTHONPATH:$HAIL_HOME/build/distributions/hail-python.zip"
```

Comment:
invert these to prepend. PYTHONPATH is stupid and can cause problems if it starts with a colon

Reply:
Hmm. Is it rude to prepend our stuff on top of what's already there?

Reply:
usually there's nothing there

Reply:
Ok I took a look at POSIX and I think we can post-pend in a PYTHONPATH safe way:
```
(hail) dking@wmb16-359 # sh
sh-3.2$ PYTHONPATH=""
sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}:$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ 
sh-3.2$ echo $PYTHONPATH
:/Users/dking/projects/hail/build/distributions/hail-python.zip
sh-3.2$ PYTHONPATH=""
sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ 
sh-3.2$ echo $PYTHONPATH
/Users/dking/projects/hail/build/distributions/hail-python.zip
sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ echo $PYTHONPATH
/Users/dking/projects/hail/build/distributions/hail-python.zip:/Users/dking/projects/hail/build/distributions/hail-python.zip
sh-3.2$ unset PYTHONPATH
sh-3.2$ echo $PYTHONPATH

sh-3.2$     export PYTHONPATH="${PYTHONPATH:+$PYTHONPATH:}$HAIL_HOME/build/distributions/hail-python.zip"
sh-3.2$ echo $PYTHONPATH
/Users/dking/projects/hail/build/distributions/hail-python.zip
```
(POSIX parameter substitution)[http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_02]